lib/commands/log.js

```diff
@@ -155,6 +154,42 @@ export async function getLogTypes() {
   return nativeLogTypes;
 }
 
+/**
+ * https://w3c.github.io/webdriver-bidi/#event-log-entryAdded
+ *
+ * @template {import('node:events').EventEmitter} EE
+ * @this {import('../driver').AndroidDriver}
+ * @param {EE} logEmitter
+ * @param {BiDiListenerProperties} properties
+ * @returns {[EE, LogListener]}
+ */
+export function assignBiDiLogListener (logEmitter, properties) {
+  const {
+    type,
+    context = NATIVE_WIN,
+    srcEventName = 'output',
+    entryTransformer,
+  } = properties;
+  const listener = (/** @type {import('../utils').LogEntry} */ logEntry) => {
+    const finalEntry = entryTransformer ? entryTransformer(logEntry) : logEntry;
+    this.eventEmitter.emit(BIDI_EVENT_NAME, {
+      context,
+      method: 'log.entryAdded',
+      params: {
+        type,
+        level: finalEntry.level,
+        source: {
+          realm: util.uuidV4(),
```

Comment:
this will be recalculated on every log emission?

Reply:
guess i'm not quite sure of the meaning of this `realm` attribute

Reply:
I've just copied the structure from the [spec](https://w3c.github.io/webdriver-bidi/#types-log-logentry). Maybe I understand it wrong

Reply:
yikes I have no idea what a realm is I guess. Could we just define it once in `assignBiDiLogListener` rather than on every log emission?

Reply:
I've just set it to an empty string for now